doc/en/user/source/production/container.rst

```diff
@@ -52,4 +52,29 @@ For more information about JVM configuration, see the article `Performance tunin
       uintx InitialHeapSize   := 268435456     {product}
       uintx MaxHeapSize       := 4294967296    {product}
     
-   The above results (from a 16 GB laptop) amount to initial heap size of 256m, and a max heap size of around 4 GB (or around 1/4 of system memory).
\ No newline at end of file
+   The above results (from a 16 GB laptop) amount to initial heap size of 256m, and a max heap size of around 4 GB (or around 1/4 of system memory).
+   
+Enable CORS
+-----------
+
+The standalone distributions of GeoServer use an application server include the Jetty application server. Enable Cross-Origin Resource Sharing (CORS) to allow JavaScript applications outside of your own domain to use GeoServer.
+
+# Add the following configuration to :file:`webapps/geoserver/WEB-INF/web.xml`:
```

Comment:
Could be worth adding an external link to http://www.eclipse.org/jetty/documentation along the lines of "For more information on what this does and other options see http://www.eclipse.org/jetty/documentation"